changelog/core_sync_event.dd

```diff
@@ -0,0 +1,34 @@
+New module core.sync.event
+
+This module provides a primitive for lightweight signaling of other threads
+by emulating Windows events on posix. It can be used to start execution of
```

Comment:
* Perhaps I'm picky, but now it sounds like it's for Posix only. I would write something like: "... a cross-platform interface  for lightweight signaling of other threads. "
* `posix` -> `Posix`

Reply:
No problem ;-) Amended.